bundle/src/main/java/com/adobe/acs/commons/fam/impl/ActionManagerImpl.java

```diff
@@ -52,6 +52,7 @@
  * Manages a pool of reusable resource resolvers and injects them into tasks
  */
 class ActionManagerImpl extends CancelHandler implements ActionManager, Serializable {
+
     private static final long serialVersionUID = 7526472295622776150L;
 
     transient private static final Logger LOG = LoggerFactory.getLogger(ActionManagerImpl.class);
```

Comment:
But TBH I'd rather just change that outside of the PR since the PR didn't introduce that code per se.